### PR TITLE
Cleaned up formatting in template_tests.test_custom.

### PR DESCRIPTION
**Pull request by prestontimmons**
This limits lines to 120, updates to direct imports, and splits the tests into a few test case.

### CODE REVIEW

File: tests/template_tests/test_custom.py [hunk 1]
```diff
@@ -96,130 +100,115 @@ def test_simple_tag_registration(self):
...
-
-        t = template.Template('{% load inclusion %}{% inclusion_no_params_with_context %}')
-        self.assertEqual(t.render(c), 'inclusion_no_params_with_context - Expected result (context value: 42)\n')
-
-        t = template.Template('{% load inclusion %}{% inclusion_params_and_context 37 %}')
-        self.assertEqual(t.render(c), 'inclusion_params_and_context - Expected result (context value: 42): 37\n')
-
-        t = template.Template('{% load inclusion %}{% inclusion_two_params 37 42 %}')
-        self.assertEqual(t.render(c), 'inclusion_two_params - Expected result: 37, 42\n')
+        msg = ("'simple_tag_without_context_parameter' is decorated with "
```
Comment (timgraham) [inline tests/template_tests/test_custom.py]: slightly prefer this style so lines don't have some non-multiple of 4 indent:
```
msg = (
   "..."
)
```

File: tests/template_tests/test_custom.py [hunk 2]
```diff
@@ -96,130 +100,115 @@ def test_simple_tag_registration(self):
...
+              "takes_context=True so it must have a first argument of 'context'")
+        contents = '{% load custom %}{% simple_tag_without_context_parameter 123 %}'
+        six.assertRaisesRegex(self, TemplateSyntaxError, msg, Template, contents)
```
Comment (timgraham) [inline tests/template_tests/test_custom.py]: I think you can use `with self.assertRaisesMessage` equivalently here (context manager form is much easier to read IMO)

**Comment by prestontimmons** [comment]
Thanks. I made those changes. I agree they're better.

FYI - I think I triggered an intermediate build before my final push.

**Comment by timgraham** [comment]
merged in 23c4cbc0fa8217fa5fa2f6ce77437d403ea0eaff, thanks.

**Pull request closed without merging**